exonum-java-binding/common/src/main/java/com/exonum/binding/common/proofs/list/ListProofHashCalculator.java

```diff
@@ -116,18 +136,16 @@ private HashCode visitLeft(ListProofBranch branch, long parentIndex) {
   }
 
   /**
-   * Returns calculated root hash of a list proof tree.
-   *
-   * @return hash code
+   * Returns calculated hash of a list proof tree.
```

Comment:
The documentation is still incomplete: is it a merkle root hash of the tree, or a hash of the ProofList collection?